src/Service/PartyHelper.php

```diff
@@ -209,4 +209,38 @@ function ($relation) {
 
         return ['complainants' => $complainants, 'counterparties' => $counterparties];
     }
+
+    public function getSortingRelevantComplainant(CaseEntity $case): string
+    {
+        $complainantTypes = $this->getComplainantPartyTypesByCase($case);
+        // Gets first entry in array
+        $relevantType = reset($complainantTypes);
+
+        $complainantRelation = $this->relationRepository
+            ->findOneBy([
+                'case' => $case,
+                'type' => $relevantType,
+                'softDeleted' => false,
+            ])
+        ;
+
+        return $complainantRelation ? $complainantRelation->getParty()->getName() : ' ';
```

Comment:
Returning a single space looks fishy. Why not return `null`?

Reply:
Returned `null` and added checks for this where this function is called.